scripts/release.py

```diff
@@ -105,7 +105,7 @@ def changelog(version, write_out=False):
 
 def main():
     init(autoreset=True)
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(allow_abbrev=False)
```

Comment:
idk why we changed this in this file, there's no plugins involved here and prefix matching is actually kind of a nice feature

Reply:
See the mentioned issues for the problems that prefix matching brings, namely that we can't know that a prefix matches a given option during preparsing, because later on plugins might register options that actually match the option given by the user.

Reply:
Oh but you got a point about changing it in this file, here doesn't really hurt. 😁

Reply:
to be clear, I agree with the change in pytest proper, just not in these helper scripts

Reply:
Yeah, I understand and agree. 👍

Reply:
ok if I add a commit to this PR which undoes those two?

Reply:
Fine by me!